tests/test_eccv_conference.py

```diff
@@ -1655,3 +1656,41 @@ def test_secondary_assignments(self, conference, client, test_client, selenium,
         secondary_assignments_div = selenium.find_element_by_id('secondary-papers')
         for i in range(1,3):
             assert secondary_assignments_div.find_elements_by_id('note-summary-{}'.format(i))
+
+    def test_chronological_pc_timeline(self, conference, selenium, request_page):
+        # Assumptions: 1) Items are in list elements
+        #              2) Dates are stated in plain English
+        def convert_text_to_datetime(text):
+            mnt_to_idx = {month.lower(): index for index, month in enumerate(calendar.month_abbr) if month}
```

Comment:
calendar.month_abbr has "Sep" for september while the one displayed in ui is "Sept"
so ln 1673 would fail
![image](https://user-images.githubusercontent.com/60613434/132573614-463fb1b6-6f83-4ca2-81fe-0e753b17d9ef.png)